src/components/button/button.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
       notes: 'Primary buttons should be used for the principle call to action on the page.',
       context: {
         variant: 'primary',
+        iconOnly: true,
```

Comment:
Seems that this variable is for including icon only button instead of turning the template to icon-only. That said, `hasIconOnly`?

Reply:
sounds good to me